includes/admin/donors/class-donor-table.php

```diff
@@ -171,11 +171,27 @@ public function column_cb( $donor ) {
 	 * @return string
 	 */
 	public function column_name( $donor ) {
-		$name     = ! empty( $donor['name'] ) ? $donor['name'] : '<em>' . __( 'Unnamed Donor', 'give' ) . '</em>';
```

Comment:
@kakshak Do not insert gravatar inside an anchor tag. It is not valid HTML structure.

![image](https://user-images.githubusercontent.com/1784821/47492801-6e64cc00-d86b-11e8-9bef-a5dd2b821ba7.png)

Reply:
@ravinderk I've put that inside `<a>` tag because Devin mentioned in `Acceptance Criteria` that `Avatars are clickable to go to the donor details screen on the donor listing screen`. Let me know your thoughts on this.